frontend/apps/marketing/src/contentful/register-custom-components.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * This file is used to register custom react components with the Contentful SDK.
```

Comment:
Does each new component need to be manually added to this file when we want it to be available on Studio/Experiences?

Reply:
Yes, only if we want it to show up Contentful Studio Experiences. Updated the comment to clarify